static/src/javascripts/projects/common/modules/experiments/tests/tailor-survey.js

```diff
@@ -118,27 +118,28 @@ const renderQuickSurvey = () => {
     // look for a survey with this ID to return. This is useful as we can easily see how a particular survey
     // would be rendered, without actually putting it live. If this parameter is empty or not specified, tailor
     // behaves as usual.
-    const surveyToShow = local.get('surveyToShow');
+    const surveyToShow = localStorage.surveyToShow || [];
```

Comment:
To support older browsers, we use the `lib/storage` module as a facade for `window.localStorage`. Was this causing troubles for you?